docs/requirements.txt

```diff
@@ -0,0 +1,18 @@
+iopath>=0.1.8
+nlpaug==1.1.3
+numpy>=1.19.5
+Pillow>=8.2.0
+python-magic>=0.4.22
+regex>=2021.4.4
+vidgear>=0.2.4
+audioread>=2.1.9
+ffmpeg-python>=0.2.0
+librosa>=0.8.1
+opencv-python>=4.5.2.54
+SoundFile>=0.10.3.post1
+torch>=1.9.0
+torchaudio>=0.9.0
+sphinx
+sphinx-epytext
+sphinx-autodoc-typehints
+myst-parser

```

Comment:
instead of copying/pasting this from the `requirements.txt` on the base level of this repo, is there any way we could populate this dynamically from the original requirements?

this will allow for less human error/forgetting to update dependencies in both locations

Reply:
+1 strongly agree!

Reply:
@jbitton There is a possible route but I'm not too sure if it will work. So basically we could require the ReadTheDocs build to run `setup.py` and within that file, the `docs/requirements.txt` is dynamically generated.

Reply:
I'm personally okay with that. As long as we only do this for readthedocs builds :)